examples/03_seedrl_swebench.py

```
@@ -0,0 +1,649 @@
+                        break
+
+                # Update stats
+                batch_size = len(batch)
+                self.total_batches += 1
+                self.max_batch_size_seen = max(self.max_batch_size_seen, batch_size)
+
+                # Run batched inference in a thread to avoid blocking the event loop
+                responses = await asyncio.to_thread(self._run_batched_inference, batch)
+
+                # Distribute responses back to the correct futures
+                for req, response in zip(batch, responses):
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)